### PR TITLE
[Merged by Bors] - feat: Extending from `Ici a`

### PR DESCRIPTION
**Pull request by YaelDillies**
https://github.com/leanprover-community/mathlib/pull/18795

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Data/Set/Intervals/ProjIcc.lean [hunk 1]
```diff
@@ -4,22 +4,30 @@ Released under Apache 2.0 license as described in the file LICENSE.
...
 import Mathlib.Data.Set.Function
-import Mathlib.Data.Set.Intervals.Basic
+import Mathlib.Data.Set.Intervals.OrdConnected
 
 /-!
 # Projection of a line onto a closed interval
 
 Given a linearly ordered type `α`, in this file we define
 
+* `Set.projIci (a : α)` to be the map `α → [a, ∞[` sending `]-∞, a]` to `a`, and each point
```
Comment (eric-wieser) [inline Mathlib/Data/Set/Intervals/ProjIcc.lean:19]: Don't we normally use `[a, ∞)` not `[a, ∞[`?

**Review by eric-wieser — APPROVED** [review]
bors d+

Thanks!

File: Mathlib/Data/Set/Intervals/ProjIcc.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/Data/Set/Intervals/ProjIcc.lean:19]: Uh sure, but that's not what the mathlib docstring uses.

**Comment by YaelDillies** [comment]
bors merge

**Pull request closed without merging**